Java: Update README for local maven installation instructions (#1905)
valkey-io/valkey-glide: java/README.md

_Note: Publishing to Maven is pending; the examples below will be available once artifacts are published. You can also download artifacts from the [GitHub release](https://github.com/valkey-io/valkey-glide/releases/tag/v1.0.0) and add the jar file to the project._

> You can publish the release to local maven by downloading the artefacts from the latest [release](https://github.com/valkey-io/valkey-glide/releases). You can update to local maven using the following example for osx-aarch64:

Reviewer: ```suggestion
**Temporary Workaround Until Artifacts are Available in Maven Central**
You can publish the release to local maven by downloading the artifacts from the latest [release](https://github.com/valkey-io/valkey-glide/releases). You can update to local maven using the following example for osx-aarch64:
```

Author: Clarification added under 980ac8e177b89a2c2e7e9167c4d9b9fd62721fa4

```bash
jar xf bundle-java-aarch64-apple-darwin.jar

> mvn install:install-file -Dfile=valkey-glide-1.0.0-osx-aarch_64.jar -DgroupId=io.valkey -DartifactId=valkey-glide -Dversion=1.0.0 -Dpackaging=jar -DPom=valkey-glide-1.0.0.pom

Reviewer: ```suggestion
mvn install:install-file -Dfile=valkey-glide-1.0.0-osx-aarch_64.jar -DgroupId=io.valkey -DartifactId=valkey-glide -Dversion=1.0.0 -Dpackaging=jar -DpomFile=valkey-glide-1.0.0.pom
```

Author: Thanks for the update

```

You can then add the valkey-glide dependency via gradle:
```gradle
repositories {

>     // Use Maven Central for resolving dependencies.

Reviewer: ```suggestion
```

Author: fixed under 980ac8e177b89a2c2e7e9167c4d9b9fd62721fa4

_Note: Publishing to Maven is pending; the examples below will be available once artifacts are published. You can also download artifacts from the [GitHub release](https://github.com/valkey-io/valkey-glide/releases/tag/v1.0.0) and add the jar file to the project._

> **Temporary Workaround while Artifacts pending to Maven Central**

Reviewer 2: Do we need this PR? GLIDE is already published on maven central.

```bash
unzip java-aarch64-apple-darwin.zip

> jar xf bundle-java-aarch64-apple-darwin.jar

Reviewer 2: I think first 2 steps are not needed anymore

Reviewer 2: 3rd line worked on my PC - I skipped those 
```
mvn install:install-file -Dfile=bundle-java-x86_64-unknown-linux-gnu.jar -DgroupId=io.valkey -DartifactId=valkey-glide -Dversion=1.0.0 -Dpackaging=jar -DPom=valkey-glide-1.0.0.pom
```

Author: I think you have to unzip

Author: pointing directly to the bundle is a good idea

Reviewer 2: no zips uploaded to the release anymore

mvn install:install-file -Dfile=valkey-glide-1.0.0-osx-aarch_64.jar -DgroupId=io.valkey -DartifactId=valkey-glide -Dversion=1.0.0 -Dpackaging=jar -DPom=valkey-glide-1.0.0.pom
```

> You can then add the valkey dependency via gradle:

Reviewer 3: ```suggestion
You can then add the Valkey dependency via gradle:
```

Author: changed to `valkey-glide`...
